doc/user/layouts/partials/create-source/connector/kafka/with-options.html

```diff
@@ -9,6 +9,7 @@
 `timestamp_frequency_ms`| `int` | Default: `1000`. Sets the timestamping frequency in `ms`. Reflects how frequently the source advances its timestamp. This measure reflects how stale data in views will be. Lower values result in more-up-to-date views but may reduce throughput.
 `topic_metadata_refresh_interval_ms` | `int` | Default: `30000`. Sets the frequency in `ms` at which the system checks for new partitions. Accepts values [0,3600000].
 `enable_auto_commit` | `boolean`| Default: `false`. Controls whether or not Materialize commits read offsets back into Kafka. This is purely for consumer progress monitoring and does not cause Materialize to resume reading from where it left off across restarts.
+`fetch_message_max_bytes` | `int` | Default: `134217728`.  Accepts values [1, 1000000000], controls the initial maximum number of bytes per topic+partition to request when fetching messages from the broker. If the client encounters a message larger than this value it will gradually try to increase it until the entire message can be fetched.
```

Comment:
Can you pattern this after one of the other lines here? e.g.

> `topic_metadata_refresh_interval_ms` | `int` | Default: `30000`. Sets the frequency in `ms` at which the system checks for new partitions. Accepts values [0,3600000].

Reply:
I moved the other one which was mid-line to the end of the line as well so all three are formatted the same way.